content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -2750,39 +2768,13 @@ private User getUser(String userid) {
 		return user;
 	}
 
-	/**
-	 * Return a Map containing all users of x type enrolled on a site
-	 * @param siteId Sakai site ID
-	 * @param role specific user role e.g. 'instructor'
-	 * @return
-	 */
-	@SuppressWarnings("unchecked")
-	private Map getAllUsers(String siteId, String role) {
-		String ROLE = "section.role."+role;
-		Map<String,String> users = new HashMap<>();
-		Site site = null;
-		try {
-			site = siteService.getSite(siteId);
-			Set<String> instIds = site.getUsersIsAllowed(ROLE);
-			List<User> activeUsers = userDirectoryService.getUsers(instIds);
-			for (int i = 0; i < activeUsers.size(); i++) {
-				User user = activeUsers.get(i);
-				users.put(user.getId(),user.getId());
-			}
-		} catch (IdUnusedException e) {
-			log.error("Unable to fetch site in getAllUsers: " + siteId, e);
-		} catch (Exception e) {
-			log.error("Exception in getAllUsers", e);
-		}
-		return users;
-	}
 	/**
 	 * The primary method of this class. Syncs the enrollment between a Sakai
 	 * Site and it's corresponding
 	 *
 	 * @param sakaiSiteID
 	 */
-	private boolean syncSiteWithTurnitin(String sakaiSiteID) {
+	private boolean syncSiteWithTurnitin(final String sakaiSiteID) {
```

Comment:
if `final` needed here?

Reply:
not really needed just helps me when doing refactoring as the IDE then lets me know if the value is being manipulated